androidbrowserhelper/src/androidTest/AndroidManifest.xml

```diff
@@ -16,7 +16,28 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="android.support.customtabs">
     <uses-sdk android:minSdkVersion="16"/>
-    <application>
+    <queries>
```

Comment:
Now that we've got everything working - are we sure we need these?

Reply:
Good point, they are not needed anymore